### PR TITLE
ISSUE-707 Label/get method

### LINKED ISSUE #707: Label/get JMAP method
Implement the /get method for Label defined in https://github.com/linagora/tmail-backend/issues/702

Request:
  ```json
  {
    "using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:labels"],
    "methodCalls": [
      [
        "Label/get",
        {
          "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
          "ids": ["123456", "notFound"]
        },
        "c1"
      ]
    ]
  }
  ```

  Response:
  ```json
  {
    "sessionState": "2c9f1b12-b35a-43e6-9af2-0106fb53a943",
    "methodResponses": [
      [
        "Label/get",
        {
          "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
          "state": "2c9f1b12-b35a-43e6-9af2-0106fb53a943",
          "list": [{
            "id": "123456",
            "displayName": "Custom Tag 1",
            "keyword": "custom_tag_1",
            "color": "#000000"
          }],
          "notFound": [
            "notFound"
          ]
        },
        "c1"
      ]
    ]
  }
  ```

Notes:
- ids `null` returns all Labels
- properties should be supported

DoD: integration tests

### PR DESCRIPTION
**Pull request by quantranhong1999**
resolve #707

### CODE REVIEW

File: tmail-backend/apps/distributed-es6-backport/src/main/java/com/linagora/tmail/james/app/DistributedServer.java [hunk 1]
```diff
@@ -241,7 +244,8 @@ public class DistributedServer {
                 new DistributedEmailAddressContactEventModule(),
                 new RabbitMQEmailAddressContactModule(),
                 new RspamdModule(),
-                new TasksHeathCheckModule());
+                new TasksHeathCheckModule(),
+                new CassandraLabelRepositoryModule());
```
Comment (chibenwa) [inline tmail-backend/apps/distributed-es6-backport/src/main/java/com/linagora/tmail/james/app/DistributedServer.java:248]: Alphabetic ordering?

**Review by chibenwa — COMMENTED** [review]
Otherwise good to me.

File: tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LabelGetMethodContract.scala [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+
+  @Test
+  def shouldReturnLabelCapabilityInSessionRoute(): Unit = {
+    val response: String = `given`()
+      .when()
+      .get("/session")
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .extract()
```
Comment (vttranlina) [inline tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LabelGetMethodContract.scala:51]: `.body("capabilities", hasKey("com:linagora:params:jmap:messages:vault"))` another pretty syntax

File: tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LabelGetMethodContract.scala [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+         |						"id": "${label1.id.id.value}",
+         |						"displayName": "Label 1",
+         |						"keyword": "${label1.keyword.flagName}",
+         |						"color": "${RED.value}"
+         |					}]
+         |			},
+         |			"c1"
+         |		]
+         |	]
+         |}""".stripMargin)
```
Comment (vttranlina) [inline tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LabelGetMethodContract.scala:505]: Should we need one more test case for get another acocuntId but NOT delegated?

**Comment by vttranlina** [comment]
Cassandra impl was merged, pls rebase it

**Review by Arsnael — COMMENTED** [review]
Looks good to me. Rebasing on that for my PR is gonna be a headache though, as we have code in common :)

**Comment by quantranhong1999** [comment]
solved comments and rebased

**Review by vttranlina — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Pull request merged**